app/models/patient.rb

```diff
@@ -23,7 +24,7 @@ def full_name
   end
 
   def to_bundle
-    entries = [json] + immunizations.map(&:json)
+    entries = ([min_json] + immunizations.map(&:min_json)).map { |e| FHIR::Bundle::Entry.new(resource: e) }
```

Comment:
Building a bundle seems like something that should be in its own class rather than being Patient's responsibility.